src/stores/GroupStore.js

```diff
@@ -110,6 +110,12 @@ export default class GroupStore extends EventEmitter {
         this._fetchSummary();
         this._fetchRooms();
         this._fetchMembers();
+
+        return {
```

Comment:
i'm a bit surprised that one can return a object like this from registerListener - is this just ignorance, or can we have a comment to explain what API this comes from?

Reply:
or is this adhoc of our own devising? in which case, it really should be jsdoc'd on the method as the return type

Reply:
This is fairly similar to the `flux/utils.Store` way of doing things, which returns `{ stop: () => {...} }`

Reply:
ok. jsdoc would be good enough, imo: "This method returns an object which callers can use to invoke an unregister hook to..." or whatever.

Reply:
Awesome, I did both in the end 👍